src/languages/en-US/guilds.json

```
@@ -0,0 +1,10 @@
+{
+	"explicitContentFilterDisabled": "Disabled",
+	"explicitContentFilterMembersWithoutRoles": "Members without roles",
+	"explicitContentFilterAllMembers": "All members",
+	"messageNotificationsAll": "All Messages",
+	"messageNotificationsMentions": "Only @mentions",
```

```suggestion
	"messageNotificationsMentions": "Only @mentions",
```

Does this only get used in embeds or in raw text as well? Because in case of the latter if some server has a role called `mentions` then won't Discord transform it into a mention like it does when typing `@skyra` in SDS?

---

Embeds only, and no, to ping a user or a role you need to use `<@{user.id}>` or `<@&{role.id}>` respectively.